src/main/scala/org/clulab/timenorm/scate/TemporalNeuralParser.scala

```diff
@@ -246,7 +246,18 @@ class TemporalNeuralParser(modelStream: Option[InputStream] = None) extends Auto
     }
   }
 
-  def inferLinks(timeSpans: Array[(Int, Int, String)]): Array[Array[(String, Int)]] = {
+  def filterBetween(property: String, text: String, source: (Int, Int, String), target: (Int, Int, String)): Boolean = {
+    val start_indicators = Array("from", "since")
+    val end_indicators = Array("to", "until")
```

Comment:
It would be better to have these hard-coded English values in a text file like your date-types.txt.